src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -13,6 +11,7 @@
  *      specifically for feedback response comments.
  */
 public class InstructorFeedbackResultsPageResponseCommentUiTest extends BaseUiTestCase {
+    private InstructorFeedbackResultsPage resultsPage;
 
     @Override
     protected void prepareTestData() {
```

Comment:
Can use `@BeforeMethod`
```java
    @Override
    protected void prepareTestData() {
        // see prepareData()
    }

    @BeforeMethod
    private void prepareData() {
        testData = loadDataBundle("/InstructorFeedbackResultsPageResponseCommentUiTest.json");
        removeAndRestoreDataBundle(testData);
    }
```

So you don't need to add `prepareTestData()` in each `@Test`